client/src/components/NewBlockForm.tsx

```diff
@@ -388,11 +391,15 @@ export class NewBlockFormPresentational extends React.Component<any, any> {
   };
 }
 
+const mapStateToProps = state => ({
+  numBlocksInRedux: state.blocks.blocks.length,
+});
+
 const mapDispatchToProps = dispatch => ({
   resetBlock: ({ id }) => dispatch(resetBlock({ id })),
 });
 
 export const NewBlockForm = connect(
-  undefined,
```

Comment:
Curious how we think about `undefined` versus `null`. Notice we use `null` ~284 times and `undefined` ~33 times.

Reply:
Yeah good question... Different people having different preferences. Might be something to keep an eye on in future PRs. Let me know if you have a preference for consistency.

Reply:
Looked into this a bit. I think we should use `null`.
![image](https://user-images.githubusercontent.com/1892071/63611309-7f898100-c5d3-11e9-97b6-ec7f14658663.png)
It seems in idiomatic javascript, when assigning an empty property value, null is preferred over undefined.
https://stackoverflow.com/questions/5076944/what-is-the-difference-between-null-and-undefined-in-javascript